config/locales/en.yml

```diff
@@ -2443,6 +2443,12 @@ en:
     If CORS is enabled, these are the origins that are allowed to access OpenProject API.
     <br/>
     Please check the <a href="%{origin_link}" target="_blank">Documentation on the Origin header</a> on how to specify the expected values.
+  setting_apiv3_max_page_size: "Maximum API page size"
+  setting_apiv3_max_page_instructions_html: >
+    Set the maximum page size the API will respond with.
+    It will not be possible to perform API requests that return more values on a single page. 
+    <br/>
+    <strong>Warning:</strong> Setting this to a high value will result in significant performance impacts.
```

Comment:
We should add a warning that setting it below the max of the pagination size will lead to errors.